libs/gl-client-py/glclient/tls.py

```diff
@@ -11,7 +11,6 @@ def __init__(self):
         # through the rust library (streaming calls)
         self.inner = native.TlsConfig()
         self.ca: Optional[bytes] = None
-        self.id: Tuple[Optional[bytes], Optional[bytes]] = (None, None)
```

Comment:
This id field is accessed by some tests from `gltesting/tests/test_gl_node.py` They are necessary there in order to set up a custom gprc connection. I thing the custom grpc connection is not needed anymore and actually not the right way to access greenlight.
I can add a fix on top of your commits if you like, as I wrote the tests that need the id field in the first place.

Reply:
That would be great